ee/modules/610-service-with-healthchecks/docs/README.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "The service-with-healthchecks module"
+---
+
+Options when pods contain many processes (like VMs) and each of them can handle traffic independently. For this purpose, it is necessary to be able to evaluate the ability of these processes to accept incoming traffic. For this purpose we have implemented an extended set in the new ServiceWithHealthchecks resource. Implementation of the module with an extended version of the standard Service resource. The specification of the ServiceWithHealthchecks resource is extended with a set of healthchecks in comparison with the standard one. At this stage http and tcp probes are implemented.
```

Comment:
It is necessary to improve the general description and add a brief explanation of what components the module consists of and how they interact.